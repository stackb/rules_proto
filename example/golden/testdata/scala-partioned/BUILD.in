exports_files(["config.yaml"])

# gazelle:proto_language scala enabled true
# gazelle:proto_rule proto_scala_library flag --partition_services
# gazelle:proto_rule grpc_scala_library flag --partition_services
