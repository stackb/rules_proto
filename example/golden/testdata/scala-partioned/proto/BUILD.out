load("@rules_proto//proto:defs.bzl", "proto_library")
load("@build_stack_rules_proto//rules/scala:proto_scala_library.bzl", "proto_scala_library")
load("@build_stack_rules_proto//rules:proto_compile.bzl", "proto_compile")

proto_library(
    name = "proto_proto",
    srcs = ["complete.proto"],
    visibility = ["//visibility:public"],
)

proto_compile(
    name = "proto_scala_compile",
    outputs = ["proto_scala.srcjar"],
    plugins = ["@build_stack_rules_proto//plugin/scalapb/scalapb:protoc-gen-scala"],
    proto = "proto_proto",
)

proto_scala_library(
    name = "proto_scala_library",
    srcs = ["proto_scala.srcjar"],
    visibility = ["//visibility:public"],
    deps = [
        "@com_google_protobuf//:protobuf_java",
        "@maven_scala//:com_thesamet_scalapb_lenses_2_12",
        "@maven_scala//:com_thesamet_scalapb_scalapb_runtime_2_12",
        "@maven_scala//:com_thesamet_scalapb_scalapb_runtime_grpc_2_12",
        "@maven_scala//:io_grpc_grpc_api",
        "@maven_scala//:io_grpc_grpc_protobuf",
        "@maven_scala//:io_grpc_grpc_stub",
    ],
)
